go-controller/pkg/ovn/address_set_sync.go

```diff
@@ -0,0 +1,424 @@
+package ovn
```

Comment:
not really, because it needs the knowledge of all the address set IDs used by ovn (like `getEgressIPAddrSetDbIDs`)

Reply:
But that's just a helper method. We could just have those in `libovsdbops` if we are concerned. 

If we end up using object ids everywhere we need to export he knowledge of how to build the ids if needed because we cannot expect to end up with a single package for everything.

Reply:
moved it to a new `address_set_syncer` package